SocialCareCaseViewerApi/V1/UseCase/CaseStatusesUseCase.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.Factories;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.Domain;
+
+namespace SocialCareCaseViewerApi.V1.UseCase.Interfaces
+{
+    public class CaseStatusesUseCase : ICaseStatusesUseCase
+    {
+        private IDatabaseGateway _databaseGateway;
+
+        public CaseStatusesUseCase(IDatabaseGateway databaseGateway)
+        {
+            _databaseGateway = databaseGateway;
+        }
+
+        public ListCaseStatusesResponse ExecuteGet(long personId)
+        {
+            var caseStatus = _databaseGateway.GetCaseStatusesByPersonId(personId);
```

Comment:
Could be worth validating the personId actually matches a person, I think this should throw a different exception than the one on line 26

Reply:
Changed the exception check and text to return "Person not found"